exchanges/gateio/gateio.go

```diff
@@ -196,58 +197,59 @@ func (g *Gateio) GetSpotKline(arg KlinesRequestParams) ([]*KLineResponse, error)
 	var rawKlines map[string]interface{}
 	err := g.SendHTTPRequest(urlPath, &rawKlines)
 	if err != nil {
-		return nil, err
+		return kline.Item{}, err
+	}
+
+	result := kline.Item{
+		Exchange: g.Name,
 	}
 
-	var result []*KLineResponse
 	if rawKlines == nil || rawKlines["data"] == nil {
-		return nil, fmt.Errorf("rawKlines is nil. Err: %s", err)
+		return kline.Item{}, fmt.Errorf("rawKlines is nil. Err: %s", err)
```

Comment:
This wasn't caused by this PR, but no `err` can exist here. So running Thrasher's example code will result with ` ./gctcli.exe gethistoriccandles gateio btc-usdt spot
2020/07/07 08:01:09 rpc error: code = Unknown desc = rawKlines is nil. Err: %!s(<nil>)`
Could you remove the err?